learnable/src/components/DeleteUser/DeleteUser.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useNavigate } from "react-router-dom";
+import "./DeleteUser.css";
+
+function DeleteUser({id}) {
+    const navigate = useNavigate();
+
+    const handleDelete = (event) => {
+        event.preventDefault();
+
+        deleteUser().then((response) => {
+            window.localStorage.removeItem("token");
+            navigate("/login");
+        });
```

Comment:
Since you don't use `response` you can remove it from the `then` call like so: 

```suggestion
        deleteUser().then(() => {
            window.localStorage.removeItem("token");
            navigate("/login");
        });
```